workflow/snakemake_rules/chores.smk

```diff
@@ -0,0 +1,38 @@
+# Copied from the monkeypox repo¹, with RSV-specific edits.
+# ¹ https://github.com/nextstrain/monkeypox/blob/5c461dc7e90cd70c1f16b193f82fd1666d4c95e2/workflow/snakemake_rules/chores.smk
+
+rule update_example_data_wildcards:
+    """This updates the files under example_data/ based on latest available data from data.nextstrain.org.
+
+    The subset of data is generated by an augur filter call which:
+    - sets the subsampling size to 50
```

Comment:
To reviewers: Does anyone know if there are filter options to better subsample the RSV data? The [monkeypox filter](https://github.com/nextstrain/monkeypox/blob/5c461dc7e90cd70c1f16b193f82fd1666d4c95e2/workflow/snakemake_rules/chores.smk#L26-L27) includes the root and groups by clade + lineage. I'm not sure if those apply here.

Reply:
Unfortunately I'm not seeing 'clade', 'lineage' or similar in the metadata:

```
$ head -n1 data/*/metadata.tsv                             
==> data/a/metadata.tsv <==
accession	genbank_accession_rev	strain	date	region	country	division	location	host	date_submitted	sra_accession	abbr_authors	reverse	authors	institution	F_coverage	G_coverage	genome_coverage

==> data/b/metadata.tsv <==
accession	genbank_accession_rev	strain	date	region	country	division	location	host	date_submitted	sra_accession	abbr_authors	reverse	authors	institution	F_coverage	G_coverage	genome_coverage
```

which would be required for the monkeypox-like filter. Clades aren't currently part of [ingest rules](https://github.com/nextstrain/rsv/tree/master/ingest/workflow/snakemake_rules), and doesn't happen until [here](https://github.com/nextstrain/rsv/blob/master/workflow/snakemake_rules/clades.smk) in the build. Maybe a way forward is adding clade assignment in ingest but I would imagine this as a separate discussion and PR.

To get at your "better subsampling" question, maybe group by year and country? Similar to the rsv config file?

https://github.com/nextstrain/rsv/blob/ba171f4a43110382c38b6154be3febd50408d7bf/config/configfile.yaml#L16-L17

Others, feel free to chime in.

Reply:
Thanks, that's a good suggestion! Updated to use that config directly in 538442d0f7ce34a748f19a672e0888f52f4786d8.

Reply:
Nice! LGTM :D